api/common.js

```diff
@@ -94,6 +94,10 @@ export const unwrapRegionalGroups = regionalGroups => ({
 })
 
 export const unwrapTeamMember = (teamMember) => {
+  if (!teamMember) {
+    return undefined
+  }
+
   const { fields } = teamMember
   return {
     id: teamMember && teamMember.sys && teamMember.sys.id,
```

Comment:
Now that we're  returning early if `!teamMember`, we shouldn't have the `teamMember && ...` part of these checks anymore.

Reply:
I guess you could say in theory that  if `teamMember.sys.id` wouldn't exist it would break. But since it is coming from contentful they should always exist.